### DISCUSSION TITLE
Issues with deploying threejs vite app

### DISCUSSION THREAD

**Discussion by JRH89** [community/community#44654] [Other Feature Feedback, Questions, & Ideas] [answered]
I have successfully deployed it but when I view the page, however the threejs elements don't show and I only see the basic HTML.

**Comment by Matteo-it** [comment, marked as answer]
Can you post your html code and vite version?

**Reply by JRH89** [reply]
Hello Matteo!
Thank you for reaching out to assist me. 
I successfully deployed the github page, it seems that my issue was with the vite.config.js file not existing, so I created one with the
basic configuration:
// vite.config.js
export default {
  // config options
}
and then added the line- base: /reponame/ 
 as seen in the Vite documentation. BTW I am using Vite version 9.3.0 and Vanilla js
